src/pages/Tasks/containers/TasksContainer/index.js

```diff
@@ -36,30 +42,40 @@ const TasksContainer = () => {
   const handleEditingTaskCancel = (taskId) => {
     dispatch(editingTaskCancel(taskId));
   };
-  const handleEditingTaskConfirm = (taskId, newText) => {
-    dispatch(editingTaskConfirm({ taskId, newText }));
+  const handleEditingTaskConfirm = (taskId, taskValues) => {
+    dispatch(editingTaskConfirm({ taskId, taskValues }));
   };
 
   return (
     <TasksView
-      task={inputText}
-      handleTaskChange={handleTextEditing}
+      task={form}
+      handleTaskChange={handleChange}
       handleTaskCreate={handleTaskCreate}
     >
-      {[...tasks].reverse().map(({ id, text, isDone, isEditing }) => (
-        <Task
-          text={text}
-          key={id}
-          id={id}
-          isDone={isDone}
-          isEditing={isEditing}
-          handleRemoveTask={handleRemoveTask}
-          handleCompletedTask={handleCompletedTask}
-          handleEditingTask={handleEditingTask}
-          handleEditingTaskCancel={handleEditingTaskCancel}
-          handleEditingTaskConfirm={handleEditingTaskConfirm}
-        />
-      ))}
+      {[...tasks]
+        .reverse()
```

Comment:
rework this logic to selector. Actually your tasks are no sorted by isCompleted field